src/components/TitleSubtitle/TitleSubtitle.css

```diff
@@ -0,0 +1,33 @@
+.section-title-black {
+    text-align: center;
+    display: block;
+    color: var(--color-text);
+    margin: 0 0 20px 0;
+    font-weight: bold;
```

Comment:
De font-weight 600 is eigenlijk standaard voor alle titels in dit project, dus het is beter om die dan bij de algemene font-declaraties te zetten in `index.css`